app/models/responses/FlashKeys.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.responses
+
+object FlashKeys {
```

Comment:
👍

Reply:
Why not use TypeClass for those ? In most cases we don't store multiple items with same type in Flash ?

Reply:
Can you show me an example?

Reply:
```
case class LRN(value: String)

object LRN {
   implicit val flashBindalbe = FlashBindable[LNR]("LRN", _.value)
}
```
On request site
```
.flashing(FlashFor(lnr))
````

Reply:
I agree that it'd be nice to make it type safe but Flash is a play class which doesn't seem to have any binder interface as far as I can see